runtime-sdk/src/modules/accounts/mod.rs

```diff
@@ -521,6 +521,17 @@ impl module::AuthHandler for Module {
                 .add(&tx.auth_info.fee.amount);
 
             // TODO: Emit event that fee has been paid.
+
+            // Bump transaction priority.
```

Comment:
Should this prioritize based on gas price instead of fee directly?

Reply:
Yeah i guess that way it doesn't prioritize higher gas amount transactions? (although maybe it should?). Anyway, changed to use gas price instead.